### PR TITLE
Fix Documentation around MaximumPayloadSizeBytes

### PR DESCRIPTION
**Pull request by stafot**
#### Summary
Fix Documentation around MaximumPayloadSizeBytes

#### Ticket Link
https://mattermost.atlassian.net/browse/CLD-8427

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thank you, @stafot!!

File: source/configure/web-server-configuration-settings.rst [hunk 1]
```diff
@@ -705,22 +705,22 @@ Cluster log timeout
...
-| For example, the upload attachment API or the API to      |                                                                                  |
-| upload a custom emoji.                                    |                                                                                  |
-|                                                           |                                                                                  |
-| Numerical value. Default is **300000** (300 kB).          |                                                                                  |
-+-----------------------------------------------------------+----------------------------------------------------------------------------------+
++-----------------------------------------------------------+-------------------------------------------------------------------------------------+
+| The maximum payload size in bytes for all APIs except     | - System Config path: N/A                                                           |
+| APIs that receive a file as an input.                     | - ``config.json`` setting: ``".ServiceSettings.MaximumPayloadSizeBytes: 100000",``  |
```
Comment (agnivade) [inline source/configure/web-server-configuration-settings.rst:720]: The default is still 300KB

Comment (stafot) [reply source/configure/web-server-configuration-settings.rst:720]: on mmctl local returned `100KB` so let's ensure it.

**Review by streamer45 — COMMENTED** [review]
Given it already caused some confusion (payload is a very generic word), I'd suggest we make it clear that this limit only applies to the body of (inbound) requests. In other words, it doesn't limit the response sizes.

**Review by andrleite — APPROVED** [review]

**Comment by stafot** [comment]
@streamer45 Feel free to follow up with another PR that improves wording on this.

**Pull request merged**